### PR TITLE
gas-oracle: L1 base fee max/min change

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Add new config to prevent changes in the L1 base fee value
in the `OVM_GasPriceOracle` from being too volatile. New config
option `--max-percent-change-l1-base-fee` expects a float and
will cap upwards or downwards the amount that the base fee can
change between two consecutive updates. This is useful for preventing
too much volatility in the L1 portion of the fee that is paid on
L2.

**Metadata**
- Fixes  ENG-1408

### CODE REVIEW

**Comment by tynes** [comment]
This is not a blocker for the devnet but will be nice to have

File: go/gas-oracle/flags/flags.go [hunk 1]
```diff
@@ -99,7 +99,13 @@ var (
 		Name:   "significant-factor",
 		Value:  0.05,
 		Usage:  "only update when the gas price changes by more than this factor",
-		EnvVar: "GAS_PRICE_ORACLE_SIGNIFICANT_FACTOR",
+		EnvVar: "GAS_PRICE_ORACLE_L2_GAS_PRICE_SIGNIFICANT_FACTOR",
+	}
+	MaxPercentChangeL1BaseFeeFlag = cli.Float64Flag{
+		Name:   "max-percent-change-l1-base-fee",
+		Value:  0.10,
+		Usage:  "max change of the base fee between 2 updates, a float that represents a percent",
```
Comment (smartcontracts) [inline go/gas-oracle/flags/flags.go:107]: does this default represent 10% or 0.1%?

Comment (tynes) [reply go/gas-oracle/flags/flags.go:107]: It represents 10%. It follows the same way other config in this application works

Comment (smartcontracts) [reply go/gas-oracle/flags/flags.go:107]: Plz just add something to the description of the config value that reflects this

**Review by smartcontracts — APPROVED** [review]
Ok I'll approve this. Feel free to address my nit but won't block merging because of it.

**Comment by tynes** [comment]
This is only useful if the polling interval is greater than the blocktime because the basefee can only move at most 12.5% each block

**Comment by tynes** [comment]
Closing this due to it going stale, we don't really need it now

**Pull request closed without merging**